docs/ppp/iteration_3_01112022.md

```diff
@@ -0,0 +1,21 @@
+### Plans 📆
+
+- Build a Lightning Full node using LDK and Flutter and open a channel automatically [https://github.com/itchysats/10101/issues/10](https://github.com/itchysats/10101/issues/10).
+- Build the trade view based on the [wireframes](https://www.figma.com/file/zUBJpQ0v8cx6DHivS10OeI/TenTenOneWireFrame).
+- Add a custom output to a Lightning Channel. This will be the base for adding DLCs to a channel. [https://github.com/itchysats/10101/issues/54](https://github.com/itchysats/10101/issues/54)
+
+### Progress ✅
+
+- Added an on-chain wallet powered by [BDK](https://github.com/bitcoindevkit/bdk) to our app. We generate a wallet basde on a Bip39 seed phrase [https://github.com/itchysats/10101/issues/68](https://github.com/itchysats/10101/issues/68).
+- Show balance in the wallet. [https://github.com/itchysats/10101/issues/90](https://github.com/itchysats/10101/issues/90).
```

Comment:
Here is a screenshot if you want to add it but it does not look like much yet:

![image](https://user-images.githubusercontent.com/5557790/199188814-c1d879cc-acbe-412b-8bd2-a0efa6b29af1.png)

Once the navigation bars and FABs (floating action buttons) are in it will look more like an app :)